server/services/datacommons.py

```diff
@@ -331,7 +331,7 @@ def get_interesting_places(dcids):
 def get_statvar_groups(dcid):
     url = API_ROOT + API_ENDPOINTS['get_statvar_groups']
     req_json = {
-        'place': dcid,
+        'places': dcid,
```

Comment:
Did you run into this issue when you run your local server?

Reply:
oh, sorry for the issue!